README.md

```diff
@@ -177,12 +175,21 @@ submitting a pull request with new or updated fixtures, we ask that contributors
 update them to use the default service ID by running `make fix-fixtures` with
 `FASTLY_TEST_SERVICE_ID` set to the same value used to run your tests.
 
+If you are **updating** an existing resource, in order to force go-vcr to re-record,
+rather than use the existing recorded values, you will have to removed the recorded
+values for that resource via `rm -r fastly/fixtures/<resource_name>`. **If you don't
+remove the existing recorded values, you will see the following error**:
+
+```
+Post "https://api.fastly.com/service/SERVICEID/version": Requested interaction not found
+```
```

Comment:
@phamann this is a major currently-undocumented "gotcha".

Reply:
Agreed, a good addition.

Reply:
Yeah, I know I've spent hours trying to debug this by diving into the backend, only to find out that I needed to remove the fixtures.